public/pages/main/Main.tsx

```diff
@@ -18,7 +18,7 @@ import React from 'react';
 import { AppState } from '../../redux/reducers';
 import { CreateDetector } from '../createDetector';
 import { DetectorList } from '../DetectorsList';
-import { ListRouterParams } from '../DetectorsList/List/List';
+import { ListRouterParams } from '../DetectorsList/Containers/List/List';
```

Comment:
We are using different naming for container folder, `Containers `(2, Dashboard and DetectorList) and `containers `(6).
How about we change to `containers` ? Similar to `Components` folder.

Reply:
Thanks, will change